### PR TITLE
[INDY-2131] Test fixtures hardening

### PR DESCRIPTION
**Pull request by andkononykhin**
- makes logic of write/read checks more accurate
- fixes pytest warning for custom marker
- dumps env of in-docker tests run

### CODE REVIEW

File: system/utils.py [hunk 1]
```diff
@@ -280,25 +280,53 @@ def check_no_failures(hosts):
...
+    return res
+
 
+async def ensure_pool_performs_write_read(
+    pool_handle, wallet_handle, trustee_did, nyms_count=1, timeout=30
+):
+    start = time.perf_counter()
+    writes = await eventually(
+        check_pool_performs_write, pool_handle, wallet_handle, trustee_did, nyms_count=nyms_count
+    )
```
Comment (skhoroshavin) [inline system/utils.py]: I think `eventually` here is not needed

File: system/utils.py [hunk 2]
```diff
@@ -280,25 +280,53 @@ def check_no_failures(hosts):
...
+async def check_pool_performs_read(pool_handle, wallet_handle, submitter_did, dids):
+    res = []
+    for did in dids:
+        resp = await get_nym(pool_handle, wallet_handle, submitter_did, did)
+        assert resp['result']['seqNo'] is not None
+        res.append(resp)
+    return res
+
 
+async def ensure_pool_performs_write_read(
```
Comment (skhoroshavin) [inline system/utils.py]: `check_pool_performs_write_read`?

Comment (andkononykhin) [reply system/utils.py]: updated

File: system/utils.py (hunk 1 above)
Comment (andkononykhin) [reply system/utils.py]: updated

**Pull request merged**